contact.html

```diff
@@ -0,0 +1,264 @@
+                </div>
+              </div>
+              <div class="servicesItem">
+                <div class="services_Info">
+                  <div class="icon">
+                    <i class="bx bx-envelope-open"></i>
+                  </div>
+                  <div class="text_Info">
+                    <a href="mailto:email@example.com">email@example.com</a>
+                  </div>
+                </div>
+              </div>
+              <div class="servicesItem">
+                <div class="services_Info">
+                  <div class="icon">
+                    <i class="bx bx-phone-call"></i>
+                  </div>
+                  <div class="text_Info">
+                    <p>Giurgiului road:</p>
+                    <a href="#">0700-00-00</a>
+                  </div>
+                </div>
+              </div>
+              <div class="servicesItem">
+                <div class="services_Info">
+                  <div class="icon">
+                    <i class="bx bx-time"></i>
+                  </div>
+                  <div class="text_Info">
+                    <p>monday-friday:</p>
+                    <p>10:00-19:00</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="contact map">
+            <iframe
+              src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2850.979001403955!2d26.091978765392227!3d44.39255277910295!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40b1fe4a99cc9d53%3A0xf3b34fa89f02ef1!2s%C8%98oseaua%20Giurgiului%20109A%2C%20Bucure%C8%99ti!5e0!3m2!1sro!2sro!4v1674231160823!5m2!1sro!2sro"
+              style="border: 0"
+              allowfullscreen=""
+            ></iframe>
+          </div>
+          <div class="contact form">
+            <form>
+              <h4>contact us</h4>
+              <div class="formContainer">
+                <div class="row_left_right">
+                  <div class="inputContainer">
+                    <label class="important">first name</label>
+                    <input
+                      type="text"
+                      required
+                      placeholder="enter first name"
+                    />
+                  </div>
+                  <div class="inputContainer">
+                    <label class="important">last name</label>
+                    <input type="text" required placeholder="enter last name" />
+                  </div>
+                </div>
+                <div class="row_left_right">
+                  <div class="inputContainer">
+                    <label class="important">email</label>
+                    <input
+                      type="email"
+                      required
+                      placeholder="email@example.com"
+                    />
+                  </div>
+                  <div class="inputContainer">
+                    <label class="important">mobile</label>
```

Comment:
Label-urile trebuie fie să aibă atributu `for` care să fie id-ul unui element de formular, fie să conțină un element de formular.

În situații ca a ta, eu aș pune

```html
<label class="inputContainer">
  <span class="important">
```

Dacă faci așa, să pui și `display: block` în CSS.